main.go

```diff
@@ -3,24 +3,25 @@ package main
 import (
 	"log"
 	"os"
+	"time"
 
 	"github.com/urfave/cli/v2"
 
 	"github.com/spacelift-io/spacectl/internal/cmd/profile"
 	"github.com/spacelift-io/spacectl/internal/cmd/stack"
-	versioncmd "github.com/spacelift-io/spacectl/internal/cmd/version"
 )
 
 var version = "dev"
 
 func main() {
 	app := &cli.App{
-		Name:  "spacectl",
-		Usage: "Programmatic access to Spacelift GraphQL API",
+		Name:     "spacectl",
+		Version:  version,
+		Compiled: time.Now(),
```

Comment:
I don't think this is correct - it'll return the current time rather than the compilation time. I think what we'd need to do is something similar to the `version` variable - create a var for the compilation time and then set it via the linker.

It looks like goreleaser supports this out the box: https://goreleaser.com/customization/build/. Based on the information about the `.Date` variable [here](https://goreleaser.com/customization/templates/), I think what you'd need to do is:

- Declare a var called `date` as a string.
- Parse it using [`time.Parse()`](https://pkg.go.dev/time#Parse).

Reply:
Yeah - in a way it's actually safer to do what you've done and just re-add the version command as-is, without changing its format. That way we don't worry about it, and we can either make it consistent or drop the command completely at some point in the future.